test/test_onnx.py

```diff
@@ -0,0 +1,84 @@
+import numpy
```

Comment:
this seems to be used only for the `all_close`, can we replace it instead with `torch.testing.assert_allclose`?

Reply:
calling `.numpy()` on a torch tensor doesn't require `numpy` to be imported, so this `import` can be safely removed if we replace the `all_close` call.